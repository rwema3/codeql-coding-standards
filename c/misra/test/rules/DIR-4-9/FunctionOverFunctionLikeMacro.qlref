rules/DIR-4-9/FunctionOverFunctionLikeMacro.ql