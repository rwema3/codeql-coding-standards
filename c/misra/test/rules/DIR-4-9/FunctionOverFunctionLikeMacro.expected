| test.c:6:1:6:25 | #define MACRO4(x) (x + 1) | Macro used instead of a function. |
| test.c:11:1:11:48 | #define MACRO9() printf_custom("output = %d", 7) | Macro used instead of a function. |
